(Merge 4/22) feat(charts): add section about customizing chart thresholds (#17004)
newrelic/docs-website: src/content/docs/query-your-data/explore-query-data/use-charts/use-your-charts.mdx

  <DoNotTranslate>**[one.newrelic.com > All capabilities](https://one.newrelic.com/all-capabilities) > [Query your data](https://one.newrelic.com/data-exploration/query-builder)**</DoNotTranslate>: This is an example of a query and how to select a null value display option.
  </figcaption>
  </Collapser>

  <Collapser
    id="thresholds"
    title="Thresholds"
>

>     Thresholds are visual cues that add context to your charts and tables, making data interpretation a breeze. They work by highlighting specific ranges within your data using:

Author: ```suggestion
    Thresholds are visual cues that add context to your charts and tables, making data interpretation clearer. Here's how they can elevate your visualizations:
```
